hugo/config.toml

```diff
@@ -24,24 +24,28 @@ disableHugoGeneratorInject = true
 [menu]
   [[menu.main]]
       name = "Demo"
-      weight = 5
+      weight = 10
       url = "https://demo.gardener.cloud"
   [[menu.main]]
       name = "Adopters"
-      weight = 10
+      weight = 20
       url = "/adopter"
   [[menu.main]]
       name = "Documentation"
-      weight = 20
+      weight = 30
       url = "/docs"
   [[menu.main]]
       name = "Blogs"
-      weight = 30
+      weight = 40
       url = "/blog"
   [[menu.main]]
       name = "Community"
-      weight = 40
+      weight = 50
       url = "/community"
+  [[menu.main]]
+      name = "Join us on"
+      weight = 60
+      url = "https://demo.gardener.cloud"
```

Comment:
This doesn't look right

Reply:
Thank you for spotting this. The Slack link works correctly when testing, which is probably why this mistake managed to evade me.